lib/manageiq/cross_repo/runner.rb

```diff
@@ -110,7 +111,7 @@ def build_test_script
       commands += sections.flat_map do |section|
         # Travis sections can have a single command or an array of commands
         section_commands = Array(travis_yml[section]).map { |cmd| "#{cmd} || exit $?" }
-        next if section_commands.blank?
+        next if section_commands.empty?
```

Comment:
think this will remove the need for the ugly trailing `compact`

```suggestion
        next([]) if section_commands.empty?
```

Reply:
Changes logic more than I would like for removing a dependency.  Can be done if a followup if deemed important.